iron-Quotes/src/main/java/com/theironyard/controllers/IronQuotesController.java

```diff
@@ -0,0 +1,109 @@
+package com.theironyard.controllers;
+
+import com.theironyard.command.QuoteCommand;
+import com.theironyard.command.TagCommand;
+import com.theironyard.entities.Quote;
+import com.theironyard.entities.Tag;
+import com.theironyard.services.QuoteRepository;
+import com.theironyard.services.TagRepository;
+import com.theironyard.services.UserRepository;
+import com.theironyard.utilities.PasswordStorage;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Collection;
+import java.util.List;
+
+
+/**
+ * Created by EddyJ on 8/1/16.
+ */
+@RestController
+public class IronQuotesController {
+
+    @Autowired
+    UserRepository userRepository;
+
+    @Autowired
+    QuoteRepository quoteRepository;
+
+    @Autowired
+    TagRepository tagRepository;
+
+    @RequestMapping(path = "/quotes", method = RequestMethod.POST)
+    public Quote createQuote(@RequestBody Quote quote){
+        quoteRepository.save(quote);
+        return quote;
+    }
+
+    @RequestMapping(path = "/quotes",method = RequestMethod.GET)
+    public List<Quote> getAllQuotes(){
+        List<Quote> quoteList = quoteRepository.findAll();
+        return quoteList;
+    }
+
+    @RequestMapping(path = "/quotes/{id}",method = RequestMethod.GET)
+    public Quote getQuote(@PathVariable Integer id){
+        Quote quote = quoteRepository.findOne(id);
+        return quote;
+    }
+
+    @RequestMapping(path = "/quotes/{id}",method = RequestMethod.PUT)
+    public Quote updateQuote(@PathVariable Integer id, @RequestBody QuoteCommand quoteCommand){
+        Quote quote = quoteRepository.findOne(id);
+        quote.setText(quoteCommand.getText());
+        quote.setAuthor(quoteCommand.getAuthor());
+        quoteRepository.save(quote);
+        return quote;
+    }
+
+    @RequestMapping(path = "/quotes/{id}",method = RequestMethod.DELETE)
+    public void deleteQuote(@PathVariable Integer id) throws Exception {
+        Quote deletedQuote = quoteRepository.findOne(id);
+        if(deletedQuote == null){
+            throw new Exception("Quote does not exist");
```

Comment:
This should be a custom exception with the response annotation.  By simply throwing the exception you are going to serve back a 500 error to the client which is not what actually happened.  This should return a 400 Bad Request because the client sent an invalid id.